docs/examples-placeholders.md

```diff
@@ -19,38 +19,37 @@ tags:
 
 ## Use meaningful examples
 
-- Make sure example and placeholder text is always meaningful and matches real use cases
-- Avoid exposing real addresses or proprietary information through examples
+- Make sure examples are meaningful and matches real use cases
+- Avoid exposing real data or proprietary information through examples
+- Format examples as *italics*, but also text that the user is expected to replace or enter
+- Introduce the examples you're about to show and explain what they allow to do
 
 **:thumbsup: Correct**
 
-user@domain.com
+*michael.scott@dundermifflin.com*
 
 **:thumbsdown: Don't use**
 
-bobama@whitehouse.com
+*bobama@whitehouse.com*
 
 handle@domain.tld
 
-## Format standard text as *italics*
+## When examples are code
 
-Format example text and text the user is expected to replace or enter as *italics*.
+When examples that the user is expected to replace occurs in a block of code, do the following:
 
-## How to deal with sample code
+- make it all caps. For example, *ACCOUNT*
+- Format it as *italics*
+- separate words with underscores. For example, *ACCOUNT_ID*
+- Do not use other symbols like `$`, or `{}`
+- After the sample code, instruct the user to replace placeholder values
+
```

Comment:
The listing is not consistent, we start bullet lists with a Capital letter, here we have a mix